### PR TITLE
LPS-76552 Convert UserRolesFormNavigatorEntry.java to a ScreenNavigationEntry

### PR DESCRIPTION
**Pull request by pei-jung**
Move section "Roles" to General > Roles

screen shot 2018-01-09 at 11 32 21 am

/cc @drewbrokke

### CODE REVIEW

**Comment by pei-jung** [comment]
ci:test

File: modules/apps/foundation/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/portlet/action/UpdateRolesMVCActionCommand.java [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+				actionResponse.setRenderParameter("mvcPath", "/error.jsp");
+			}
+			else if (e instanceof MembershipPolicyException) {
+				SessionErrors.add(actionRequest, e.getClass(), e);
+
+				actionRequest.setAttribute(
+					UsersAdminWebKeys.ACTION_COMMAND_NAME,
+					"/users_admin/update_roles");
+				actionRequest.setAttribute(
+					UsersAdminWebKeys.JSP_PATH, "/user/roles.jsp");
```
Comment (drewbrokke) [inline modules/apps/foundation/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/portlet/action/UpdateRolesMVCActionCommand.java]: Is it necessary to set these two attributes if we're already setting the screen navigation parameters below?  Won't their classes take care of it?

Comment (pei-jung) [reply modules/apps/foundation/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/portlet/action/UpdateRolesMVCActionCommand.java]: That's right. We don't need these two attributes set. I'll make the change and update the branch. Thanks for catching it!

Comment (drewbrokke) [reply modules/apps/foundation/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/portlet/action/UpdateRolesMVCActionCommand.java]: 👍

**Comment by drewbrokke** [comment]
@pei-jung looks like some jsp errors on this one.

**Comment by pei-jung** [comment]
ci:retest

Fixed jsp error and removed unnecessary attributes.

**Comment by pei-jung** [comment]
Rebased and resent https://github.com/alee8888/liferay-portal/pull/525.

**Pull request closed without merging**